packages/gluestick-plugin-bunyan/src/server.js

```diff
@@ -19,24 +20,41 @@ export type Logger = {
   error: Function,
 };
 
+type PluginUtilities = {
+  requireModule: Function,
+  defaultLogger: Logger,
+};
+
 const defaultSettings: Options = {
   name: 'default name',
 };
 
-const bunyanPlugin = (): { logger: Logger } => {
-  let options: Options = defaultSettings;
+const bunyanPlugin = (
+  opts: Object,
+  { requireModule, defaultLogger }: PluginUtilities,
+): { logger?: Logger } => {
+  let options: Options = { ...defaultSettings };
   try {
-    // $FlowIgnore
-    const userSettings: Options = require('src/bunyan.config.js').default;
+    const userSettings: Options = requireModule(
+      path.join(process.cwd(), 'src/bunyan.config.js'),
```

Comment:
I've gotten to like the aliases because of the automatic typing it gives us, though there's no reason to change this right now.